lib/glific/messages.ex

```diff
@@ -141,6 +141,19 @@ defmodule Glific.Messages do
     %Message{}
     |> Message.changeset(attrs)
     |> Repo.insert()
+    |> case do
+      {:ok, message} ->
+        Communications.publish_data(
```

Comment:
We don't need the subscription here? since this will be handled by the sent_message subscription.